libraries/stdlib/genmdl/cm.mdl

```diff
@@ -0,0 +1,144 @@
+//
+//   TM & (c) 2020 Lucasfilm Entertainment Company Ltd. and Lucasfilm Ltd.
+//   All rights reserved.  See LICENSE.txt for license.
+//
+//   Default color management transforms for MDL
+//
+mdl 1.5;
+using core import *;
+import::base::*;
+import::anno::*;
+import::state::*;
+import.::swizzle::*;
+import::math::*;
```

Comment:
Seems to be missing a space between `import` and the module to import (not sure if that is valid syntax).

Reply:
I didn't notice that. I copied this off of Lutz's version. It does compile properly.